### ISSUE TITLE
MISO LMP data

### ISSUE THREAD

**Issue by evanlipton** [kmax12/gridstatus#166]
In the get_lmp function for MISO, I do not see where the date variable is being used. When I try run the function I get the following error:
`    
TypeError                                 Traceback (most recent call last)
Cell In[85], line 4
      1 import gridstatus
      2 iso = gridstatus.MISO()
----> 4 df = iso.get_lmp(date = "latest",market= "DAY_AHEAD_HOURLY")
      5 print(df)

File ~/PycharmProjects/gridstatus_forked/gridstatus/lmp_config.py:26, in lmp_config.__call__.<locals>.wrapper(*args, **kwargs)
     24 if len(args) > 0 and isinstance(args[0], ISOBase):
     25     bound_args = self._verify_bound_args(bound_args)
---> 26     return self._class_method_wrapper(func, bound_args)
     27 else:
     28     # This is a runtime check after method is called.
     29     # Possible improvement: move this to "compile"-time.
     30     raise ValueError("Must be class method on ISOBase")

File ~/PycharmProjects/gridstatus_forked/gridstatus/lmp_config.py:61, in lmp_config._class_method_wrapper(func, bound_args)
     59 instance_args = bound_args.args
     60 instance_kwargs = bound_args.kwargs
---> 61 return func(*instance_args, **instance_kwargs)

File ~/PycharmProjects/gridstatus_forked/gridstatus/miso.py:150, in MISO.get_lmp(self, date, market, locations, verbose)
    148 time = r["LMPData"]["RefId"]
    149 time_str = time[:11] + " " + time[-9:-4]
--> 150 time_zone = time[-3:]
    151 time = (
    152     pd.to_datetime(time_str)
    153     .tz_localize(
   (...)
    156     .tz_convert(self.default_timezone)
    157 )
    159 if market == Markets.REAL_TIME_5_MIN:

File /usr/local/lib/python3.8/site-packages/pandas/_libs/tslibs/timestamps.pyx:1728, in pandas._libs.tslibs.timestamps.Timestamp.tz_localize()

TypeError: Cannot localize tz-aware Timestamp, use tz_convert for conversions
`

**Comment by kmax12** [comment]
this is the same issue in #163 and was fixed in #164. Let me do another release of the library. I'll let you know when it's ready to upgrade.

**Comment by kmax12** [comment]
release is up. you can upgrade to 0.19.0 with

```
pip install --upgrade gridstatus
```
or
```
pip install gridstatus==0.19.0
```
